### PR TITLE
Thanks page refresh

### PR DESCRIPTION
**Pull request by Scollaco**
# What

ThanksPage refresh.

# Why

https://trello.com/c/Wn2Faoo6/299-more-refresh-thanks-page

simulator screen shot - iphone x - 2017-12-15 at 13 22 29

simulator screen shot - iphone x - 2017-12-15 at 13 22 32

GIF

### CODE REVIEW

**Comment by dannyalright** [comment]
Is the final string `Share this project`?

**Comment by Scollaco** [comment]
@dannyalright Yes,  but I have another PR in the kickstarter repo with the `See all [category_name] projects` string waiting for approval. After that I will update the screenshot.

**Comment by justinswart** [comment]
@Scollaco you taking a look into the failing screenshots?

**Comment by Scollaco** [comment]
@justinswart Yes, but I was waiting for a new string to be deployed so I could change a button title and update the snapshots. that's why I didn't try to fix it immediately .

File: Kickstarter-iOS/Library/Nib.swift [hunk 1]
```diff
@@ -3,8 +3,10 @@ import UIKit
 public enum Nib: String {
   case BackerDashboardEmptyStateCell
```
[the remark this answers is not in the document]
Comment (Scollaco) [reply Kickstarter-iOS/Library/Nib.swift]: I think we are using the names capitalised so we can use the raw value to register the nib file here: ```self.register(UINib(nibName: nib.rawValue, bundle: bundle), forCellReuseIdentifier: nib.rawValue)```

[the remark this answers is not in the document]
Comment (justinswart) [reply Kickstarter-iOS/Library/Nib.swift]: Ah good point!

**Comment by Scollaco** [comment]
[More refresh: thanks page](https://trello.com/c/Wn2Faoo6/299-more-refresh-thanks-page)

**Pull request merged**